bundles/org.openhab.binding.yamahamusiccast/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,236 @@
+			<channel id="channelSoundProgram" typeId="channelSoundProgram"/>
+			<channel id="channelSelectPreset" typeId="channelSelectPreset"/>
+			<channel id="channelSleep" typeId="channelSleep"/>
+			<channel id="channelMCServer" typeId="channelMCServer"/>
+			<channel id="channelUnlinkMCServer" typeId="channelUnlinkMCServer"/>
+			<channel id="channelRecallScene" typeId="channelRecallScene"/>
+			<channel id="channelVolumeAbs" typeId="channelVolumeAbs"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="zone4Controls">
+		<label>Zone 4</label>
+		<channels>
+			<channel id="channelPower" typeId="channelPower"/>
+			<channel id="channelMute" typeId="channelMute"/>
+			<channel id="channelVolume" typeId="channelVolume"/>
+			<channel id="channelInput" typeId="channelInput"/>
+			<channel id="channelSoundProgram" typeId="channelSoundProgram"/>
+			<channel id="channelSelectPreset" typeId="channelSelectPreset"/>
+			<channel id="channelSleep" typeId="channelSleep"/>
+			<channel id="channelMCServer" typeId="channelMCServer"/>
+			<channel id="channelUnlinkMCServer" typeId="channelUnlinkMCServer"/>
+			<channel id="channelRecallScene" typeId="channelRecallScene"/>
+			<channel id="channelVolumeAbs" typeId="channelVolumeAbs"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="playerControls">
+		<label>player Controls</label>
+		<channels>
+			<channel id="channelPlayer" typeId="channelPlayer"/>
+			<channel id="channelArtist" typeId="channelArtist"/>
+			<channel id="channelTrack" typeId="channelTrack"/>
+			<channel id="channelAlbum" typeId="channelAlbum"/>
+			<channel id="channelAlbumArt" typeId="channelAlbumArt"/>
+			<channel id="channelRepeat" typeId="channelRepeat"/>
+			<channel id="channelShuffle" typeId="channelShuffle"/>
+		</channels>
+	</channel-group-type>
+
+
+
+
+	<channel-type id="channelPower">
+		<item-type>Switch</item-type>
+		<label>Power</label>
+		<description>Power channel</description>
+	</channel-type>
+	<channel-type id="channelMute">
+		<item-type>Switch</item-type>
+		<label>Mute</label>
+		<description>Mute channel</description>
+	</channel-type>
+	<channel-type id="channelVolume">
+		<item-type>Dimmer</item-type>
+		<label>Volume</label>
+		<description>Volume channel - percentage</description>
+		<state min="0" max="100"/>
+	</channel-type>
+	<channel-type id="channelVolumeAbs">
+		<item-type>Number</item-type>
+		<label>Volume</label>
+		<description>Volume channel - Absolute value</description>
+	</channel-type>
+	<channel-type id="channelInput">
+		<item-type>String</item-type>
+		<label>Input</label>
+		<description>Input channel</description>
+	</channel-type>
+	<channel-type id="channelSoundProgram">
+		<item-type>String</item-type>
+		<label>SoundProgram</label>
+		<description>SoundProgram channel</description>
+	</channel-type>
+	<channel-type id="channelSelectPreset">
+		<item-type>String</item-type>
+		<label>Select Net Radio/USB Preset</label>
```

Comment:
Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Please check all.

Reply:
Done